apps/omg_watcher/lib/omg_watcher/web/controllers/transaction.ex

```diff
@@ -48,8 +48,14 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
   """
   def submit(conn, params) do
     with {:ok, tx} <- expect(params, "transaction", :hex) do
+      Appsignal.increment_counter("transaction.succeed", 1)
```

Comment:
this doesn't bring us much info (this is the watcher and the endpoint is just a bypass to the child chain), but we can keep it.

Note however, that for the child chain server's counterpart, we'd probably want something more elaborate (`transaction.succeed` and `transaction.failed.<the_exact_reason_of_failure>` maybe? or more generally `transaction.failed_statelessly` vs `transaction.failed_statefully` would be an interesting diagnostic, that is, on which stage did the transaction get rejected).

Having that in mind, it could be interesting to see some tps measure of transactions successfully processed in the Watcher (e.g. when it's syncing etc). So I guess this measurement in `controllers/transaction.ex` doesn't really measure the transaction count in a very interesting way